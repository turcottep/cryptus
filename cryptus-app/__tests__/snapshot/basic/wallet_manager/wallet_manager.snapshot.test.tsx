import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import WalletManager from "../../../../components/basic/wallet_manager/wallet_manager";

describe("<WalletManager />", () => {
  it("should display properly", async () => {
    const mockProps = {
      user: {
        wallets: ["wallet1", "wallet2"],
      },
      callback_close_wallet: null,
      isMobile: false,
    };

    const tree = renderer.create(<WalletManager {...mockProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
