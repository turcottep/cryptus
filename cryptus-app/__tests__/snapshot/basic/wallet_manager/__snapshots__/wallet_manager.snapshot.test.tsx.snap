// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WalletManager /> should display properly 1`] = `
<div
  id="bg"
  onClick={[Function]}
>
  <div>
    <div>
      <div>
        <div>
          <ul>
            <div>
              <div>
                <div>
                  Metamask
                  <img
                    src="/images/metamask_logo.png"
                  />
                </div>
                <div />
              </div>
              <div>
                <img
                  onClick={[Function]}
                  src="/icons/delete_icon.png"
                />
              </div>
            </div>
            <div>
              <div>
                <div>
                  Metamask
                  <img
                    src="/images/metamask_logo.png"
                  />
                </div>
                <div />
              </div>
              <div>
                <img
                  onClick={[Function]}
                  src="/icons/delete_icon.png"
                />
              </div>
            </div>
          </ul>
        </div>
      </div>
      <div />
      <div>
        <ul />
      </div>
    </div>
  </div>
</div>
`;
